ROADMAP.rst

```diff
@@ -0,0 +1,96 @@
+* To help in the user's graduation to a "real" editor we should ensure generic editor behaviours work as per the conventions: shortcut keys are the same, editing activities (e.g. selecting and acting upon blocks of code) should work as expected and helpful features (like code completion and call tips) should work in "the usual way".
+
+Multi Device Support
+====================
+
+* Adafruit, Calliope and others have expressed an interest in Mu supporting their devices. It should, and do so automagically. It should be possible to use USB's manufacturer and device ID's to work out what's plugged in and adapt accordingly.
+* It should be obvious from the source code how to add newly supported devices. This may be related to the "Python contexts" work mentioned in the Generic Editor section above. Such work should at least encompass adding flashing a device with the appropriate version of MicroPython, copying / flashing scripts to the device and connecting to an on-board file-system and connecting to the REPL.
+
+New Features
+============
+
+* Carlos has a spike proof of concept for Blockly integration (i.e. visual programming a la Scratch - move blocks around to create a program and it emits Python). Should we implement this (and I believe many teachers would love to see this) we *must* address the problem of migrating beginners from a visual style of programming to text based programming. This could be addressed by some of the UI related points from above.
+* The REPL, while an amzingly powerful feature, can appear a little unfriendly to beginners. Jupyter notebooks are a replacement for the REPL but also allow additional annotations and images to interweave with the code to be evaluated. That they can be shared and are fully editable also means they're a great way to share a "movement of thought" in words, code and pictures. I'd like to explore how these could be used to deliver lesson plans, exercises and "follow-along" resources for teachers and learners.
+
+Documentation
+=============
+
+* We need some.
+* Read the Docs looks like a good place to start - especially since they appear to support internationalisation and can watch our repos for update.
+
+Website
+=======
+
+* Is currently a bootstrap based affair cobbled together one afternoon by me and improved at PyCon UK.
+* Installation instructions / videos.
+* "How do I?" instructions / videos.
+* I guess we need design work and a logo (I know just the person to ask: Steve Hawkes does the amazing design work for PyCon UK).
+
+Releases
+========
+
+* We should investigate packaging for Windows and OSX. Perhaps the PSF could fund the necessary developer keys needed to sign such packages.
```

Comment:
Very promising directions. I'd be surprised if the PSF were to refuse support to a great goal with proven experience at the helm.